src/boards/include/boards/cytron_maker_pi_rp2040.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_CYTRON_MAKER_PI_RP2040_H
+#define _BOARDS_CYTRON_MAKER_PI_RP2040_H
+
+// For board detection
+#define CYTRON_MAKER_PI_RP2040
+
+// --- BOARD SPECIFIC ---
+
+// Motor driver
+#ifndef MAKER_PI_RP2040_M1A_PIN
+#define MAKER_PI_RP2040_M1A_PIN 8
+#endif
+
+#ifndef MAKER_PI_RP2040_M1B_PIN
+#define MAKER_PI_RP2040_M1B_PIN 9
+#endif
+
+#ifndef MAKER_PI_RP2040_M2B_PIN
+#define MAKER_PI_RP2040_M2B_PIN 10
+#endif
+
+#ifndef MAKER_PI_RP2040_M2B_PIN
+#define MAKER_PI_RP2040_M2B_PIN 11
+#endif
+
+// Grove ports
+#ifndef MAKER_PI_RP2040_GROVE1A_PIN
+#define MAKER_PI_RP2040_GROVE1A_PIN 0
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE1B_PIN
+#define MAKER_PI_RP2040_GROVE1B_PIN 1
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE2A_PIN
+#define MAKER_PI_RP2040_GROVE2A_PIN 2
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE2B_PIN
+#define MAKER_PI_RP2040_GROVE2B_PIN 3
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE3A_PIN
+#define MAKER_PI_RP2040_GROVE3A_PIN 4
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE3B_PIN
+#define MAKER_PI_RP2040_GROVE3B_PIN 5
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE4A_PIN
+#define MAKER_PI_RP2040_GROVE4A_PIN 16
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE4B_PIN
+#define MAKER_PI_RP2040_GROVE4B_PIN 17
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE5A_PIN
+#define MAKER_PI_RP2040_GROVE5A_PIN 6
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE5B_PIN
+#define MAKER_PI_RP2040_GROVE5B_PIN 26
+#endif
+
+#ifndef MAKER_PI_RP2040_GROVE6A_PIN
+#define MAKER_PI_RP2040_GROVE6A_PIN 26
```

Comment:
You seem to have re-used pin 26 here?

Reply:
Ahhh, looks like that might have been deliberate? How strange.
[Link to the board's schematics](https://drive.google.com/file/d/1Zp8GYO8x7ThObB1G8RIZx2YdqrXtdUc0/view)